easybuild/easyblocks/c/condaenv.py

```diff
@@ -111,19 +127,3 @@ def make_module_req_guess(self):
             'MANPATH': ['man', os.path.join('share', 'man')],
             'PKG_CONFIG_PATH': [os.path.join(x, 'pkgconfig') for x in ['lib', 'lib32', 'lib64', 'share']],
         }
-
-    def install_conda_env(self):
```

Comment:
@jerowe I see no good reason to add a custom method for this? It can just be embedded in `install_step` imho